genrobots/main.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Command genrobots generates a robots.txt file for the given buckets.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+)
+
+var (
+	versionRE = regexp.MustCompile(`(rev\d+-)?v?(\d+\.\d+\.\d+|HEAD)`)
+)
+
+func main() {
+	googleapisBucketName := flag.String("gbucket", "", "googleapis.dev bucket name")
+	cgcBucketName := flag.String("cgcbucket", "", "c.g.c. bucket name")
+	robotsFilename := "robots.txt"
+
+	flag.Parse()
+
+	if *googleapisBucketName == "" {
+		fmt.Fprintf(os.Stderr, "must set -gbucket")
```

Comment:
nit: seems like you are also using log, maybe that should be does for these first couple fmt calls as well. i.e: `log.Fatal(...)`

Reply:
Done.